libs/ngx-dsfr/radio/src/lib/radio.component.html

```diff
@@ -19,10 +19,10 @@
         </label>
       </div>
     </div>
-    <p *ngIf="hasFailed" [id]="fieldSetId + '-error'" class="fr-error-text">
+    <p *ngIf="hasFailed" [id]="items[0].id + '-error'" class="fr-error-text">
       {{failureMessage}}
     </p>
-    <p *ngIf="hasSucceeded" [id]="fieldSetId + '-valid'" class="fr-valid-text">
+    <p *ngIf="hasSucceeded" [id]="items[0].id + '-valid'" class="fr-valid-text">
```

Comment:
I think you added these changes to the wrong file...
It's `radio.component.html` and not `checkbox.component.html`...

Reply:
Done